nautobot/ui/src/components/index.js

```diff
@@ -0,0 +1 @@
+// TODO: Let's make components importable from here
```

Comment:
Allow

```js
import { SidebarNav, RouterLink, LoadingWidget } from "@components";
```

instead of

```js
import SidebarNav from "@components/common/SidebarNav";
import RouterLink from "@components/common/RouterLink";
import LoadingWidget from "@components/common/LoadingWidget";
```